### ISSUE TITLE
Remove documentation?

### ISSUE THREAD

**Issue by fredrikbaberg** [CodeDocs/CodeDocs#20]
I haven't seen it documented: how can documentation be removed from CodeDocs? Will it be done through the toggle on codedocs.xyz?

**Comment by paulnovo** [comment]
Yes, just toggle it, and click 'Apply Changes'. That should remove the docs, no one should be able to access the documentation anymore

**Comment by diracsbracket** [comment]
Does not work. After toggling the switch again after some time, the previously generated documentation is still there.
